plugins/ui/statuscol.nix

```diff
@@ -43,7 +43,8 @@ helpers.neovim-plugin.mkNeovimPlugin config {
           freeformType = with types; attrsOf anything;
           options = {
             text = mkOption {
-              type = with helpers.nixvimTypes; listOf (either str rawLua);
+              type = with helpers.nixvimTypes; nullOr (listOf (either str rawLua));
```

Comment:
In this case, directly use the nixvim helper:
```nix
text = helpers.mkNullOrOption (with helpers.nixvimTypes; listOf (either str rawLua)) ''
  Segment text.
'';
```

Reply:
I thought about this, but then I would have to remove the example from the option, right?

Reply:
Yes true, I guess it is fine to leave it like that. I should stop nitpicking...